glue_plotly/html_exporters/scatter2d.py

```diff
@@ -16,45 +20,108 @@
 class PlotlyScatter2DStaticExport(Tool):
 
     icon = PLOTLY_LOGO
-    tool_id = 'save:plotly'
+    tool_id = 'save:plotly2d'
     action_text = 'Save Plotly HTML page'
     tool_tip = 'Save Plotly HTML page'
 
     def activate(self):
 
-        filename, _ = compat.getsavefilename(parent=self.viewer, basedir="plot.html")
-
-        fig = go.Figure(layout={'width': 800, 'height': 600})
+        filename, _ = compat.getsavefilename(
+            parent=self.viewer, basedir="plot.html")
+
+        width, height = np.array(self.viewer.figure.get_size_inches()*self.viewer.figure.dpi)[
+            0], np.array(self.viewer.figure.get_size_inches()*self.viewer.figure.dpi)[1]
```

Comment:
Minor stylistic thing - I think you could probably write this as just:

```python
width, height = self.viewer.figure.get_size_inches()*self.viewer.figure.dpi
```

If not, just leave as-is.
then use that for